File: QTD.SPA/src/app/_DtoModels/Version_EnablingObjective_Tool_Link/Version_EnablingObjective_Tool_Link.ts
import { Entity } from '../Entity';
import { Version_EnablingObjective } from '../Version_EnablingObjective/Version_EnablingObjective';
import { Version_Tool } from '../Version_Tool/Version_Tool';

export class Version_EnablingObjective_Tool_Link extends Entity {
  version_EnablingObjectiveId!: any;
  version_ToolId!: any;
  version_Tool!: Version_Tool;
  version_EnablingObjective!: Version_EnablingObjective;
}

File: QTD2.Domain/Entities/Core/Version_EnablingObjective_Tool_Link.cs
namespace QTD2.Domain.Entities.Core
{
    public class Version_EnablingObjective_Tool_Link : Common.Entity
    {
        public int Version_EnablingObjectiveId { get; set; }

        public int Version_ToolId { get; set; }

        public virtual Version_Tool Version_Tool { get; set; }

        public virtual Version_EnablingObjective Version_EnablingObjective { get; set; }

        public Version_EnablingObjective_Tool_Link()
        {
        }

        public Version_EnablingObjective_Tool_Link(int version_EnablingObjectiveId, int version_ToolId)
        {
            Version_EnablingObjectiveId = version_EnablingObjectiveId;
            Version_ToolId = version_ToolId;
        }
    }
}
